MSCL/source/mscl/MicroStrain/MIP/Packets/MipPacket.h

```diff
@@ -164,6 +164,7 @@ namespace mscl
         //    DESC_SET_DATA_RTK         - 0x93    - Data Packet, RTK (GNSS 3) Set
         //    DESC_SET_DATA_GNSS_4      - 0x94    - Data Packet, GNSS 4 Set
         //    DESC_SET_DATA_GNSS_5      - 0x95    - Data Packet, GNSS 5 Set
+        //    DESC_SET_DATA_SYSTEM      - 0xA0    - Data Packet, System Set
```

Comment:
Not here, but in `MipPacket::isDataPacket()` System desc set needs to be added to true case in switch.

Once added System data packets will be processed and returned in `InertialNode::getDataPackets()` - if parser isn't defined a data point will still be created with the whole payload of type Bytes (can still be printed with `as_string()`).